### PR TITLE
updating with new KFM silent policy changes

### PR DESCRIPTION
**Pull request by cartergreen**
- we support files that aren't selected for sync in the silent policy, we will just force sync those libraries 
- we support folder selection in the silent policy

### CODE REVIEW

**Comment by cartergreen** [comment]
@kaarins @gacarini

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: OneDrive/redirect-known-folders.md [hunk 1]
```diff
@@ -79,19 +79,17 @@ To check eligibility, data volume, and item counts as you decide on a rollout pl
   
 - [Silently move Windows known folders to OneDrive](use-group-policy.md#silently-move-windows-known-folders-to-onedrive)
     
-    Use this setting to redirect and move known folders to OneDrive without any user interaction. 
+    Use this setting to redirect and move known folders to OneDrive without any user interaction. Move all the folders or select the desired individual folders.  After a folder is moved, the policy will not affect the folder again, even if the selection for the folder changes.
```
Comment (JohanFreelancer9) [inline OneDrive/redirect-known-folders.md:82]:
```suggestion
    Use this setting to redirect and move known folders to OneDrive without any user interaction. Move all the folders or select the desired individual folders. After a folder is moved, the policy will not affect the folder again, even if the selection for the folder changes. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**